packages/default/scss/input/_variables.scss

```diff
@@ -72,6 +72,13 @@ $kendo-input-md-line-height: $kendo-input-line-height !default;
 /// @group input
 $kendo-input-lg-line-height: var( --kendo-line-height-lg, normal ) !default;
 
+/// The calculated height of the Input.
+/// @group input
+$kendo-input-calc-size: calc( calc( #{$kendo-input-line-height} * 1em ) + calc( #{$kendo-input-padding-y} * 2 ) + #{$kendo-input-border-width * 2} ) !default;
```

Comment:
do we need the nested calc here? The calculations should be working properly with 1 calc only.